### DISCUSSION TITLE
[BUG] The maximum `link` length of 255 is too short

### DISCUSSION THREAD

**Discussion by shaburov** [allure-framework/allure2#2186] [TestOps]
TestOps 61cb5344426a9704b7ca0fa9ead6b5d51ff93a92

![image](https://github.com/allure-framework/allure2/assets/15269131/6de33bcf-bff6-497c-9cfd-e5df1bde08ba)

![telegram-cloud-document-2-5341518068596097164](https://github.com/allure-framework/allure2/assets/15269131/e5ca4ec9-ee70-4b19-8440-e72d9d1b7a3c)

**Comment by shaburov** [comment]
UP

**Reply by cheshi-mantu** [reply]
Hi @shaburov 
For he future reference, we support Allure Testops via dedicated support portal https://help.qameta.io, all bugs, enhancement suggestions are to be reported via portal, this is the only way we can react in a timely manner.

Generally, the described behaviour is not a bug, but a limitation of the software you are using related to the data submitted to the system for storing.

There is a couple of ways you can overcome the issue you are facing right away:
- you can use a separate step|sub-step to add the link, step has no limitation
- you can use an attachment with MIME type `text/uri-list` to add long links
- you can use `text/markdown` attachment and have the same data added to a step or a test

![image](https://github.com/allure-framework/allure2/assets/8607348/55aa3c88-bfc8-4827-b866-1e0e22a73266)

**Reply by shaburov** [reply]
@cheshi-mantu I have 3k+ tests running. Sending 3k+ attachment files takes an unacceptable amount of time for us when running in CI. Empirically tested.
